### PR TITLE
callback 2

### PR DESCRIPTION
**Pull request by alex461919-test**
callback 2

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -8,19 +8,22 @@ import { login_request } from "./store/authSlice";
...
         <DatePicker />
-        <Button type="primary" style={{ marginLeft: 8 }} onClick={onLoginClick}>
+        <Button
+          type="primary"
+          style={{ marginLeft: 8 }}
+          onClick={useCallback(
+            (event) => {
+              dispatch(login_request({ user: "user", password: "password" }));
+            },
+            [dispatch]
```
Comment (alex461919) [inline src/App.tsx:24]: sure?

**Comment by alex461919-test** [comment]
> sure? 

Yes!

File: src/App.tsx (hunk 1 above)
Comment (alex461919-test) [reply src/App.tsx:24]: No!

Comment (alex461919-test) [reply src/App.tsx:24]: yes

**Pull request merged**